### ISSUE TITLE
problems while building on CI environments

### ISSUE THREAD

**Issue by agauniyal** [uWebSockets/uWebSockets#220]
I should add that regarding recent addition of meson build, due to some CI enviroments fall back to C encoding instead of unicode, they might've some problems regarding 'µWebSockets' text since 'µ' will throw errors there. There are two solutions here -

- either use 'u' in name instead of 'µ' character.
- configure utf8 locales like - 
```
export LANGUAGE=en_US.UTF-8
          export LANG=en_US.UTF-8
          export LC_ALL=en_US.UTF-8
          locale-gen en_US.UTF-8
          dpkg-reconfigure locales
```

Other builds are not affected as far as I know.

**Comment by agauniyal** [comment]
Other issues -

https://github.com/wercker/support/issues/255
https://github.com/mesonbuild/meson/issues/721

**Comment by alexhultman** [comment]
Lol just rename it to u then, it's your PR :wink:

**Comment by agauniyal** [comment]
Fixed in #222 .
